rotkehlchen/data_migrations/migrations/migration_3.py

```diff
@@ -0,0 +1,26 @@
+from typing import TYPE_CHECKING
+
+from rotkehlchen.accounting.structures import HistoryBaseEntry
+
+if TYPE_CHECKING:
+    from rotkehlchen.rotkehlchen import Rotkehlchen
+
+
+def data_migration_3(rotki: 'Rotkehlchen') -> None:
+    """
+    After changing the HistoryBaseEntry the identifiers for the events
```

Comment:
ffs this is so stupid. Why was str(asset_balance) used like that? :/

You will also need a test for this migration.

Damn I hate this. Perhaps this is completely uneeded and can wait to go in with next DB upgrade. No need to do a data migration just for a refactor.

Yeah just bundle this PR with the next DB upgrade.

Reply:
What do you mean by bundling it with the next db upgrade? Also the reason to add the asset_balance in the identifier was to avoid duplicates but is true that str(asset_balance) was not the best idea of this worl

Reply:
I mean I don't want to introduce an entire data migration for a small nice to have refactoring